### ISSUE TITLE
"interests": [] are empty

### ISSUE THREAD

**Issue by DigniticeRoland** [joshuatz/linkedin-to-jsonresume#13]
Thank you for your great work. I hope it is fine for you, when I open some issues I came across while testing. Thanks for review.

The "interests": [], is always empty, even so the profile has interest.

**Comment by joshuatz** [comment]
@DigniticeRoland Can you elaborate a bit? Right now, I'm not sure how (or if) the "interests" section on LinkedIn should map to "interests" in JSONResume schema, since they aren't exactly parallel things.

- Interests on LinkedIn are more like things you are "following", so that you get updates about them in your newsfeed (similar to following someone on Twitter, or friending someone on FB)
     - LinkedIn internally calls this data "following", not "interests" - the internal API endpoint is `.../following`
     - These have to be things that have LinkedIn pages, such as People (aka `Influencers`, e.g. `Bill Gates`), Companies (e.g. `Microsoft`), or Groups (e.g. `Microsoft Developers`)
 - Interests for JSON Resume are more like things you are passionate about, and are broad(er) categories
     - For example, `Hiking`, `Cooking`, `Social Justice`, etc.

To put it another way, saying I'm interested in `Bill Gates` on LinkedIn sounds OK, because I might want to see things related to him in my feed, but it sounds weird to say in a resume that you are "interested in Bill Gates". And on a resume, it might sound OK to say I am passionate about `Social Justice`, but that does not go in the "interests" part of my profile on LinkedIn.

It just kind of seems like LinkedIn interests might often have very little overlap with "resume interests". Does this make sense? Do you see a different possibility?

**Comment by joshuatz** [comment]
Closing due to inactivity; as I see it (without clarification), this is a non-issue at the moment, since LinkedIn's "interests" fall outside of the Resume JSON use.
